From d12b23263dd1863caf5c0831df6c26b42c97768c Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Sun, 20 Oct 2024 14:19:43 +0200
Subject: [PATCH 6/6] arm64: dts: lx2160a-clearfog-itx: enable pcie nodes for
 x4 and x8 slots

SolidRun Clearfog CX and Honeycomb have LX2160A PEX3 and PEX5 exposed on
physical connectors.
Vendor U-Boot used to patch status properties such that it went
undiscovered these nodes have their status set disabled.

Set status okay for pcie3 and pcie5 nodes.

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 .../boot/dts/freescale/fsl-lx2160a-clearfog-itx.dtsi      | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-lx2160a-clearfog-itx.dtsi b/arch/arm64/boot/dts/freescale/fsl-lx2160a-clearfog-itx.dtsi
index 41702e7386e3..f43d6a90e162 100644
--- a/arch/arm64/boot/dts/freescale/fsl-lx2160a-clearfog-itx.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-lx2160a-clearfog-itx.dtsi
@@ -96,6 +96,14 @@ &esdhc0 {
 	status = "okay";
 };
 
+&pcie3 {
+	status = "okay";
+};
+
+&pcie5 {
+	status = "okay";
+};
+
 &pcs_mdio7 {
 	status = "okay";
 };
-- 
2.43.0

